src/angularComponent/ngcOmniboxController.js

```diff
@@ -0,0 +1,11 @@
+import OmniboxController from 'src/coreComponent/OmniboxController.js';
```

Comment:
@onlywei what pattern do you want to use then to have core logic that would potentially be useful outside of Angular? I basically want Angular to be a way to build out API, and for all important logic to be in the core.